src/dialogs/vkbasalt.py

```diff
@@ -75,33 +94,13 @@ def __idle_save(self, *args):
 
         config = ManagerUtils.get_bottle_path(self.config)
 
-        class settings:
-            default = False
-            effects = False
-            output = False
-            disable_on_launch = False
-            toggle_key = False
-            cas_sharpness = False
-            dls_sharpness = False
-            dls_denoise = False
-            fxaa_subpixel_quality = False
-            fxaa_quality_edge_threshold = False
-            fxaa_quality_edge_threshold_min = False
-            smaa_edge_detection = False
-            smaa_threshold = False
-            smaa_max_search_steps = False
-            smaa_max_search_steps_diagonal = False
-            smaa_corner_rounding = False
-            lut_file_path = False
-            exec = False
-
         # Applies default settings and closes dialog.
         if self.default.get_state() is True:
-            settings.default = True
+            VkBasaltSettings.settings.default = True
```

Comment:
I think here you want to do
```python
settings = VkBasaltSettings()
settings.settings.default = True
```
python always throws me off with this kind of stuff, I am no sure if using that will alter the default in future dialogs (Open a dialog, set a setting, close the dialog and open another, will the Settings class have the default values or the last setted ones?).

Reply:
I think you got this diff from an older commit. This one works: https://github.com/bottlesdevs/Bottles/pull/1668/commits/f909b38ea63531328a7b1cb60632d02515378ed9.